modules/search/zlibrary.py

```diff
@@ -0,0 +1,109 @@
+"""
+OWASP Maryam!
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+from bs4 import BeautifulSoup
```

Comment:
use lxml instead of bs4 because lxml is pretty faster than bs4.

Reply:
This would require changes to the entire util and module codeb. Instead I have used lxml parser of beautiful soup instead of HTML parser in latest commit.